src/php/php_server.rs

```diff
@@ -32,6 +34,14 @@ impl PhpServer {
 pub(crate) fn start() -> PhpServer {
     let php_bin = binaries::get_project_version();
 
+    let phpbin_path = PathBuf::from(php_bin.as_str());
+
+    if !phpbin_path.is_executable() {
+        error!("PHP Binary not found or not executable: {}", php_bin.as_str());
+        info!("Execute rymfony php:list --refresh to solve this issue");
+        return PhpServer::new(0, PhpServerSapi::Unknown);
```

Comment:
Shouldn't you just `panic!()` instead?

Reply:
I have changed in `panic!`.

The previous behavior with the PR #69 permit Rymfony to start to serve only static file.